### ISSUE TITLE
Problem with the BERT

### ISSUE THREAD

**Issue by cloudyskyy** [eomiso/NLP_text-style-transfer#1]
Thank you with your helpful code of the paper. However, when I am  running the "bert_pretrained/classifier.py" for training on YELP dataset, the accuracy of  the model is just around 60%, why would this happen? When I train a MLP for classification, its accuracy can even reach 97%.

**Comment by eomiso** [comment]
Hi @cloudyskyy I'm sorry for the late reply. Is the accuracy only 60% after 20 epochs? How many epochs did you run?

**Comment by eomiso** [comment]
@cloudyskyy I've just run the classifier.py and you are right, the loss function and the accuracy are not getting better at all. I'm working on it, it's been a while since I've worked on this code but I believe it would be the problem with the data format. The data format might have been changed after pretraining the classifier for the style-transfer model... thus this peculiar problem occurs when trying to retrain the bert classifier.
